app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
It is advisable to indent and start writing the code from line 2 according to the pep8 standard. And leave one empty line at the end of the code file. And also place modules using brackets.

from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)